jans-cedarling/cedarling/src/authz/entities/test_create.rs

```diff
@@ -250,7 +250,8 @@ fn get_token_claim_type_boolean_error() {
         "test_id_key": "test_id",
         "string_key": "test string",
         "long_key": 1234,
-        "entity_uid_key": 123,
+        "entity_uid_key": "ff910f15-d5a4-4227-828e-11cb8463f1b7",
+        // This will trigger the type error, because it's not a bool.
```

Comment:
probably just don't forget to remove this comment before merging. it only makes sense reading from the PR.

Reply:
I deliberately left it in the PR.

My reasoning is: text fixtures like this are mostly made of placeholder data, with only one or two values that are actually used in the test. I've found it's generally a good idea to comment on those in-use values to clarify the intent. In this case, why is the in-use value `123` rather than `1572` or `true` or `false` or `"some random string"`.

Reply:
If we want to add this kind of comment. I feel like we need to add comment like this not only in `get_token_claim_type_boolean_error` but also in 
- `get_token_claim_type_string_error`
- `get_token_claim_type_long_error`
- `get_token_claim_type_entity_uid_error`
and maybe in
- `get_token_claim_cedar_typename_error`
- `get_token_claim_cedar_typename_in_attr_error`

Reply:
oh okay, i didn't realize that this was being used for a test. I can't see the whole file from the "Files changed" tab so i thought that this was just a misconfiguration of some sort.

Reply:
> If we want to add this kind of comment. I feel like we need to add comment like this not only in get_token_claim_type_boolean_error but also in ...

Sure, I can do that.

Reply:
@olehbozhok I don't yet know enough about schemas to make sensible comments for `get_token_claim_cedar_typename_error` and `get_token_claim_cedar_typename_in_attr_error`.

Perhaps you'd like to do those?